src/v2/Apps/Fair/Routes/FairArtworks.tsx

```diff
@@ -105,6 +102,7 @@ export const FairArtworksRefetchContainer = createRefetchContainer(
         @argumentDefinitions(
           acquireable: { type: "Boolean" }
           aggregations: { type: "[ArtworkAggregation]" }
+          artistIDs: { type: "[String]" }
```

Comment:
This has to be part of our queries since we're going to be filtering by one or more artists.